www/templates/components/trades/payout.template.html

```diff
@@ -16,7 +16,7 @@
     <div class="col col-33 basis" ng-show="basis == 'payout'">{{'trade.payout' | translate}}</div>
     <div class="col col-33 basis" ng-show="basis == 'stake'">{{'trade.stake' | translate}}</div>
     <div class="col col-33 amount">
-        <input type="number" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
+        <input type="text" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
```

Comment:
Problem is when user enters "1." , we sent "amount":0, so server return error.
When user enters "1.", we should send  "amount":1, then it works as expected.